pkgs/development/tools/packer/default.nix

```diff
@@ -40,7 +44,178 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    mv bin/* $out/bin
+    mv bin/* $out/bin # */
+  '';
+
+  passthru.updateScript = writeScript "update-packer" ''
+    #!${bash}/bin/bash
```

Comment:
I've tried to do it, but it wasn't able to catch all those extra deps that are specified in `vendor/vendor.json` in packer repository - resulting `.nix`-file contained only `packer` sources itself.